applications/GeoMechanicsApplication/tests/hexa_8n_normal_load/MaterialParameters.json

```diff
@@ -0,0 +1,29 @@
+{
+   "properties": [{
+      "model_part_name":         "PorousDomain.Soil",
+      "properties_id":           1,
+      "Material": {
+          "constitutive_law": {
+              "name"             :  "LinearElastic3DLaw" 
+          },
+          "Variables": {
+              "IGNORE_UNDRAINED"         :  true,
+              "YOUNG_MODULUS"            :  3.0e7,
+              "POISSON_RATIO"            :  0,
+              "DENSITY_SOLID"            :  0,
+              "DENSITY_WATER"            :  1.0e3,
+              "POROSITY"                 :  0.3,
+              "BULK_MODULUS_SOLID"       :  1.0e16,
+              "BULK_MODULUS_FLUID"       :  2.0e-30,
+              "PERMEABILITY_XX"          :  4.5e-30,
+              "PERMEABILITY_YY"          :  4.5e-30,
+              "PERMEABILITY_ZZ"          :  4.5e-30,
```

Comment:
Yet another question: these values are extremely small. Is that necessary (i.e. do we need positive values here)? Or would setting them to zero also work?

Reply:
I don't know, we don't do any pressure calculations, so I doubt it would matter. Do you have a preference @WPK4FEM?